snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotChecksum.java

```diff
@@ -61,11 +62,8 @@ private static boolean isNotMetadataFile(final Path file) {
 
   public static void persist(final Path checksumPath, final ImmutableChecksumsSFV checksum)
       throws IOException {
-    try (final RandomAccessFile checksumFile = new RandomAccessFile(checksumPath.toFile(), "rwd")) {
-      final byte[] data = checksum.serializeSfvFileData();
-      checksumFile.write(data);
-      checksumFile.setLength(data.length);
-    }
+    final FileOutputStream fileOutputStream = new FileOutputStream(checksumPath.toFile());
```

Comment:
:x: Make sure to close the output stream

Reply:
I knew I was missing something 😆 thanks good catch